### PR TITLE
TE Dockerfile refactoring

### PR DESCRIPTION
**Pull request by PinGwynn**

### CODE REVIEW

File: services/topology-engine/Dockerfile [hunk 1]
```diff
@@ -14,23 +14,12 @@
...
-RUN pip install requests
-RUN pip install gevent
-RUN pip install kazoo
-RUN pip install python-logstash
-RUN chmod +x /usr/local/bin/uwsgi
+RUN apt-get update && \
+      apt-get install -y python-pip supervisor vim && \
+      pip install --upgrade pip && \
+      pip install neo4j-driver \
+      kafka py2neo gevent python-logstash && \
```
Comment (carmine) [inline services/topology-engine/Dockerfile]: This line ends in && \  ... but there is no next line

File: services/topology-engine/Dockerfile [hunk 2]
```diff
@@ -14,23 +14,12 @@
...
-RUN pip install py2neo
-RUN pip install requests
-RUN pip install gevent
-RUN pip install kazoo
-RUN pip install python-logstash
-RUN chmod +x /usr/local/bin/uwsgi
+RUN apt-get update && \
+      apt-get install -y python-pip supervisor vim && \
+      pip install --upgrade pip && \
+      pip install neo4j-driver \
```
Comment (surabujin) [inline services/topology-engine/Dockerfile]: We don't use neo4j-driver it can be dropped too.

File: services/topology-engine/Dockerfile (hunk 1 above)
Comment (PinGwynn) [reply services/topology-engine/Dockerfile]: Fixed

File: services/topology-engine/Dockerfile (hunk 2 above)
Comment (PinGwynn) [reply services/topology-engine/Dockerfile]: Fixed

**Comment by carmine** [comment]
All suggested changes merged

**Pull request merged**